src/index.js

```diff
@@ -3,4 +3,40 @@
  * Code written here will be executed when application launches
  */
 
-console.log("...");
\ No newline at end of file
+class WeatherDataSource {
+    constructor() {
+        this.apiKey = '84af29f03ef35bb4dd7f1e9b0ad575e2';
+    }  
+    getweather(city) {

```

Comment:
- **Important:** The primary reason for its existence is to return weather data (presumably in some agreed upon form as an object of Weather or WeatherData class that has yet to be written) - instead it returns nothing at all.

Reply:
class and function return date and it's working